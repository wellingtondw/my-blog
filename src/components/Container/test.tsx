import { render } from "utils/test-utils";
import { Container } from ".";

describe("<Container />", () => {
  it("Should be able to render correctly", () => {
    const { container } = render(<Container />);

    expect(container).toMatchSnapshot();
  });
});
