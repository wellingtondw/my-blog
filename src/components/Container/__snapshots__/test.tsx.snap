// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Should be able to render correctly 1`] = `
.c0 {
  max-width: 120rem;
  margin: 0 auto;
}

@media (max-width:1170px) {
  .c0 {
    padding: 0 1.6rem;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
